docs/user/aws/install.md

```diff
@@ -48,6 +46,11 @@ In EC2, there will be 6 running instances:
 
 ![EC2 instances after install](images/install_nodes.png)
 
+The installation creates an encrypted AMI for the bootstrap and control-plane machines.
+The encrypted AMI is [copied][encrypted-copy] from the AMI configured in the control-plane machine-API provider spec, 
+which is RHCOS by default. The encryption uses the default EBS key for your target account and region 
+(`aws kms describe-key --key-id alias/aws/ebs`). The encrypted AMI is de-registered by `destroy cluster`.
```

Comment:
nit: no need for the hyphen in "de-registered" (at least [the AWS docs][1] us an unhyphenated "deregister").

[1]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/deregister-ami.html

Reply:
Fixed.